dockerfiles/platform-gnn-gpu/README.md

```diff
@@ -1,45 +1,100 @@
 
-When preparing the Dockerfile, specially for Tensorflow 2 with GPU support it is important to reference
-Tensorflow and Nvidia Cuda Dockerfiles and Docker images to pick compatible libabries.
+### GPU support in our Azure subscription
 
-### Docker image and Dockerfile Reference
+Our Azure subscription does include VM Size of GPU support. These VM Sizes normally are much more expensive than normal VM Sizes, probaly at least 10 times.  Each these VM Sizes with GPU support have different hardware and may require different driver version such as NVidia CUDA driver. Also the Tensorflow 2 and Nvidia Cuda libraries are not always compatible.
 
-#### Tensforflow
+Generally  some ND, NCv2 or NCv3 family VM Sizes should be supported in our Azure subscription
+For example following are supported: standard_nc6s_v3,standard_nc8as_t4_v3,standard_nv12s_v3,standard_nv24s_v3,standard_nv48s_v3
 
-- Dockerfile:
-  https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/dockerfiles/dockerfiles/gpu.Dockerfile
-  Above Dockeerfile is based https://hub.docker.com/layers/nvidia/cuda/. There are runime, base, etc.
-  For now we just reference https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/ and replace the package version for Ubuntu 20.04
+We current build our Docker Image with NVidia CUDA 11.4 and latest Tensorflow packages. User may need to create their own Docker Images with a different CUDA driver version and Tensorflow version.
 
-- The generated tensorflow/tensorflow:2.2.0-gpu
-  https://hub.docker.com/layers/tensorflow/tensorflow/2.2.0-gpu/images/sha256-3f8f06cdfbc09c54568f191bbc54419b348ecc08dc5e031a53c22c6bba0a252e?context=explore
+When preparing the Dockerfile, specially for Tensorflow 2 with GPU support it is important to reference
+Tensorflow and Nvidia Cuda Dockerfiles and Docker images to pick compatible libabries.
 
-#### Nvidia Cuda
+### Docker image and Dockerfile Reference
 
-- NVDA CUDA Dockerfile: https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/ubuntu20.04/
+- https://docs.nvidia.com/grid/cloud-service-support.html
+- https://docs.microsoft.com/en-us/azure/virtual-machines/linux/n-series-driver-setup
+- https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/hpccompute-gpu-linux
+- NVIDIA GPU Cloud (NGC) https://www.nvidia.com/en-us/gpu-cloud/
 
-  https://hub.docker.com/layers/nvidia/cuda/11.3.1-base/images/sha256-351d731f46159c1afdb003121e96b512a77c410c2b43ee6ea74cf1413bb858ab?context=explore
 
-- NGC-DL-CONTAINER-LICENSE
-  https://gitlab.com/nvidia/container-images/cuda/-/blob/master/LICENSE under BSD 3-Clause "New" or "Revised" License
+#### Dockerfile Consideration
+Tensorflow only provides CUDA 11.2 Dockerfile for GPU support: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/dockerfiles/dockerfiles/gpu.Dockerfile
 
+When preparing our Dockerfile (CUDA 11.4) we reference above gpu.Dockerfile with following modification
```

Comment:
When preparing our Dockerfile (CUDA 11.4) we reference the above gpu.Dockerfile with following modifications